README.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please modify README.Rmd and run `pkgdown::build_site()` to update README.md -->
+
+```{r, echo = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "README-"
+)
+```
+
+# synapserutils
+
+The `synapserutils` package provides a set of utility functions, built on top of [`synapser`](http://sage-bionetworks.github.io/synapser/) package.
+
+## Installation
+
+`synapserutils` is available as a ready-built package for Microsoft Windows and Mac OSX. For Linux systems, it is available to install from source. It can be installed or upgraded using the standard `install.packages()` command, adding the [Sage Bionetworks R Archive Network (RAN)](https://sage-bionetworks.github.io/ran) to the repository list, e.g.:
+
+```{r eval=F}
+install.packages("synapserutils", repos=c("https://sage-bionetworks.github.io/ran", "http://cran.fhcrc.org"))
+```
+Alternatively, edit your `~/.Rprofile` and configure your default repositories:
+```{r eval=F}
+options(repos=c("https://sage-bionetworks.github.io/ran", "http://cran.fhcrc.org"))
+```
+after which you may run `install.packages` without specifying the repositories:
+```{r eval=F}
+install.packages("synapserutils")
+```
+
+If you have been asked to validate a release candidate, please replace the URL https://sage-bionetworks.github.io/ran with https://sage-bionetworks.github.io/staging-ran, that is:
+
+```{r eval=F}
+install.packages("synapserutils", repos=c("https://sage-bionetworks.github.io/staging-ran", "http://cran.fhcrc.org"))
+```
+
+## Usage
+
+To get started, try logging into Synapse. If you don't already have a Synapse account, register [here](https://www.synapse.org/register):
+
+```{r eval=F}
+library(synapserutils)
+synLogin()
+```
+
+## Available Utilities
+
+### Copy
+
+#### Copy Entity (File, Folder, Table, Link, and Project)
+
+The example below copy File ID `syn123` to Project ID `syn456`:
+```{r eval=F}
+  copy("syn123", "syn456")
```

Comment:
It looks like there's some extra space here and in a couple other code chunks.

Reply:
Thanks @karawoo! Do you refer to the spaces before `copy` or the spaces between the parameters?

Reply:
The space before `copy`. I like the space between the parameters.